code/modules/clothing/clothing.dm

```diff
@@ -331,9 +337,13 @@ BLIND     // can't see anything
 						if(H.internals)
 							H.internals.icon_state = "internal0"
 						H.internal = null
-			if(flags_inv == HIDEFACE) //Means that only things like bandanas and balaclavas will be affected since they obscure the identity of the wearer.
+			if(flags_inv && HIDEFACE) //Means that only things like bandanas and balaclavas will be affected since they obscure the identity of the wearer.

```

Comment:
this would be a single `&`
it's checking for a bitflag, HIDEFACE, in the flags_inv var